src/test_merge.py

```diff
@@ -0,0 +1,39 @@
+# _*_ coding: utf-8 _*_
+"""Test the merge sort algorithm."""
+import random
+
+
+RANDOM_LIST = [random.sample(range(1000),
+               random.randrange(2, 100)) for i in range(500)]
+
+
+def test_random_lst():
+    """A."""
+    from merge_sort import merge_sort
+    for lst in RANDOM_LIST:
+        lst_sort = sorted(lst)
+        assert merge_sort(lst) == lst_sort
+
+
+def test_random_sames():
+    """A."""
+    from merge_sort import merge_sort
+    assert merge_sort([2, 2, 2, 1]) == [1, 2, 2, 2]
+
+
+def test_stability():
+    """A."""
```

Comment:
See if you can revise this test, which is not testing stability at the moment, since both items in the tuple are checked during the comparison.